packages/core/src/App/Containers/Layout/header/trading-hub-header.jsx

```diff
@@ -45,8 +45,47 @@ export const TradersHubHomeButton = ({ is_dark_mode }) => {
     );
 };
 
-const TradingHubOnboarding = ({ is_dark_mode, setIsOnboardingVisited }) => {
-    const history = useHistory();
+// the tradershub onboarding is temporarily disabled for wallets demo will enabled once everthing is completed
+// const TradingHubOnboarding = ({ is_dark_mode, setIsOnboardingVisited }) => {
+//     const history = useHistory();
+//     return (
+//         <div className='trading-hub-header__tradinghub--onboarding'>
+//             <div className='trading-hub-header__tradinghub--onboarding--logo'>
+//                 <Popover
+//                     classNameBubble='account-settings-toggle__tooltip'
+//                     alignment='bottom'
+//                     message={<Localize i18n_default_text='View onboarding' />}
+//                     should_disable_pointer_events
+//                     zIndex={9999}
+//                 >
+//                     <Icon
+//                         icon={is_dark_mode ? 'IcAppstoreTradingHubOnboardingDark' : 'IcAppstoreTradingHubOnboarding'}
+//                         size={20}
+//                         onClick={() => {
+//                             history.push(routes.onboarding);
+//                             setIsOnboardingVisited(false);
+//                         }}
+//                     />
+//                 </Popover>
+//             </div>
+//         </div>
+//     );
+// };
+
+// the tradershub onboarding is used wallets demo (temporary)
+const TradingHubOnboarding = ({ is_dark_mode, toggleIsWalletTourOpen, switchAccount, app_contents_scroll_ref }) => {
```

Comment:
@aizad-deriv you need to check of we upgraded to the banner show this and if not show the previous one.